ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Assertions.scala

```diff
@@ -41,21 +44,44 @@ object Assertions {
     }
   }
 
+  /** non-regex overload for assertGrpcError which just does a substring check.
+    */
+  def assertGrpcError(
+      t: Throwable,
+      expectedCode: Status.Code,
+      exceptionMessageSubString: String,
```

Comment:
I'd rather have it as optional and using `None` in callers when not provided. Also, please name the parameter explicitly in callers.